webapp/src/main/java/rocks/metaldetector/service/email/EmailServiceImpl.java

```diff
@@ -17,17 +18,18 @@
 
 @Service
 @Slf4j
-@AllArgsConstructor
+@RequiredArgsConstructor
 public class EmailServiceImpl implements EmailService {
 
   private final JavaMailSender emailSender;
   private final ISpringTemplateEngine templateEngine;
   private final MailProperties mailProperties;
   private final MimeMessageHelperFunction messageHelperFunction;
+  private String frontendBaseUrl;
```

Comment:
```suggestion
  @Setter
  @Value("${frontend.origin}")
  private String frontendBaseUrl;
```
So the method below is not needed?

Reply:
nice, will do in follow-up pr.